src/Controller/SlackController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Controller;
+
+use App\Notifiers\Slack\SlackNotifier;
+use App\Repository\ConferenceRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class SlackController extends AbstractController
+{
+    private SlackNotifier $slackNotifier;
+    private EntityManagerInterface $em;
+    private ConferenceRepository $conferenceRepository;
+    private HttpClientInterface $httpClient;
+
+    public function __construct(SlackNotifier $slackNotifier, EntityManagerInterface $em, ConferenceRepository $conferenceRepository, HttpClientInterface $httpClient)
+    {
+        $this->slackNotifier = $slackNotifier;
+        $this->em = $em;
+        $this->conferenceRepository = $conferenceRepository;
+        $this->httpClient = $httpClient ?? HttpClient::create();
+    }
+
+    /** @Route("/slack", name="slack_endpoint") */
+    public function slackEndPoint(Request $request): Response
+    {
+        $payload = json_decode($request->request->get('payload'));
```

Comment:
usually, we use an array: 
```suggestion
        $payload = json_decode($request->request->get('payload'), true);
```